docs/vpm-spec.md

```diff
@@ -0,0 +1,433 @@
+The VPM Client fetches the package information from the VPM Repository and installs the package.
+The VPM Client acts as a User Agent in the HTTP request to the VPM Repository.
+
+The VPM Clients usually have feature to manage VPM Repository installed for the VPM Manifest folder.
+The VPM Client **MAY** have feature to add VPM Repository from Web Browsers by using the `vcc:` URL scheme.
+
+The VPM Repository is the server that provides the package information and the package archive file.
+The VPM Repository provides the package information in [[JSON]] format and the package archive file in Unity Package format.
+One single VPM Repository can be consists of multiple servers, such as the package information server and the package archive server.
+The VPM Repository acts as an HTTP server to provide the package information and the package archive file.
+
+As described above, the VPM Client and VPM Repository communicate with each other by using the HTTP protocol [[HTTP]].
+
+VPM Profile Folder is the folder that contains the configuration files and the cache files of the VPM Client.
+The configuration file for VPM Client should be shared among the VPM Clients because it contains the user-wide installed repositories list.
+However, the VPM Client **MAY** have different configuration files for some purposes, such as beta testing.
+Actually, the recent VCC Beta uses their own folder as the VPM Profile Folder.
+
+The VPM Package is the package distributed by the VPM.
+The VPM Package will be recognized as a UPM Package by Unity Editor.
+Therefore, the VPM Package is a special type of UPM Package.
+
+### 3.2. The UPM, UPM Package
+The UPM is the Unity Package Manager, which is the package manager developed by Unity Technologies and built into Unity Editor.
+The UPM provides the package management system for Unity Editor.
+
+The UPM Package is the package recognized by the UPM and Unity Editor.
+All VPM Packages are recognized as Embedded UPM Packages by Unity Editor.
+
+### 3.3. Package Manifest, VPM Project Manifest
+The Package Manifest is the JSON document that describes the package information.
+The Package Manifest is located as `package.json` in the UPM Package and provides the package information,
+such as the package name, version, dependencies, and the package archive file name.
+
+The VPM Project Manifest is the JSON document that describes the installed package information.
+The VPM Project Manifest is located as `Packages/vpm-manifest.json` in the Unity project and provides the installed package information,
+The VPM Project Manifest has two main sections, `dependencies` and `locked`.
+The `dependencies` section provides the requested package information.
+The `locked` section provides the all installed package information, including the dependencies of the requested packages.
+
+### 3.4. Requested Package, Dependency Package, Locked Package, and Unlocked Package
+The requested package means the VPM Packages that user requested to install.
```

Comment:
plurality is not consistent, use "requested package**s**"?

nit: `s/means/are`? more concise?

counterpoint: https://developers.google.com/tech-writing/one/clear-sentences#choose_strong_verbs

However it's hard to keep using variations of verbs:

> That said, a form of be is sometimes the best choice of verb, so don't feel that you have to eliminate every form of be from your writing.

I think "are" is fine here.